src/divider.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import StylePropable from './mixins/style-propable';
+import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
+import ThemeManager from './styles/theme-manager';
+
+const Divider = React.createClass({
+
+  mixins: [StylePropable],
+
+  contextTypes: {
+    muiTheme: React.PropTypes.object,
+  },
+
+  propTypes: {
+    /**
+     * If true, the divider will be indented by 72px
+     */
+    inset: React.PropTypes.bool,
+
+    /**
+     * Override the inline-styles of the list divider's root element.
+     */
+    style: React.PropTypes.object,
```

Comment:
This component is going to be exposed. We need the theme context to work on his own.
Unless you have an idea on how to make it, I would say not a good idea :grimacing:.